test/dlc_tests/ecs/pytorch/training/test_ecs_pytorch_training.py

```diff
@@ -132,7 +132,7 @@ def test_ecs_pytorch_training_dgl_cpu(
     """
     _, image_framework_version = get_framework_and_version_from_tag(pytorch_training)
     # TODO: Remove when DGL gpu test on ecs get fixed
-    if Version(image_framework_version) == Version("1.10"):
+    if Version("1.10") <= Version(image_framework_version) < Version("1.11"):
```

Comment:
Please change this to `Version(image_framework_version) in SpecifierSet("==1.10.*")`
```suggestion
    if Version(image_framework_version) in SpecifierSet("==1.10.*"):
```